### ISSUE TITLE
proseccing tool is  crashes

### ISSUE THREAD

**Issue by ginga9** [qgis/QGIS#51635]
### What is the bug or the crash?

Processing Tools was unchecked.Forced termination by checking or double-clicking.　version　3.16~3.28　
version　3 does not work.　The old version　2  works.
![114402](https://user-images.githubusercontent.com/123916014/215376881-142336bb-fda9-441f-848b-ac9a7853a901.png)

### Steps to reproduce the issue

Processing Tools was unchecked.

Forced termination by checking or double-clicking.

### Versions

version　3.16~3.28　
version　3 does not work.　The old version　2  works.

### Supported QGIS version

- [ ] I'm running a supported QGIS version according to the roadmap.

### New profile

- [ ] I tried with a new QGIS profile

**Comment by ginga9** [comment]
Deleting the registry key doesn't help either.

**Comment by nyalldawson** [comment]
This is a local issue, and you'll need to post a lot more details to assist with debugging in order to get an answer.

**Comment by ginga9** [comment]
Thank you.

I have an experimental plugin."This plugin connects GeODin with QGIS"
And it didn't start so I deleted it.
Next time I started QGIS, the check was removed from the core plugin.
And check orIt now crashes when double-clicked.
It's still the same.
I've also tried deleting the registry key, but no change.
Functions other than core plugins can be used.


It is also possible that the Python code distributed by the Japanese government was dragged into QGIS.
https://data-gov.note.jp/n/n367f1e368d22


After forcibly quitting with this, the core plugin was removed and it did not start.


Sorry for the vagueness, but this is the situation.
